### ISSUE TITLE
Precipitation amounts seem off?

### ISSUE THREAD

**Issue by 0-5-0** [davidtakac/bura#77]
The temperature is well under -10 celcius but it's forecasting rain. My local weather provider also has an alert out for 10-15cm of snow but Bura is only showing 5.6cm next 24 hours, should some of that rain be converted to additional snow?

It's an Open-Meteo issue?

<img src="https://github.com/user-attachments/assets/3e24f901-e664-491a-aa64-5b85d0cb38c0" width=45% height=45%>

**Comment by davidtakac** [comment]
Yes, this is an Open-Meteo issue. I don't question the data I get from the API much. If they say rain and snow, I display rain and snow 😄
